ObjectiveGit/GTDiffPatch.m

```diff
@@ -59,6 +59,16 @@ - (NSUInteger)sizeWithContext:(BOOL)includeContext hunkHeaders:(BOOL)includeHunk
 	return git_patch_size(self.git_patch, includeContext, includeHunkHeaders, includeFileHeaders);
 }
 
+- (NSData*)toBuffer {
+	git_buf buf = GIT_BUF_INIT_CONST(0, NULL);
+	git_patch_to_buf(&buf, self.git_patch);
+
+	NSData* buffer = [[NSData alloc] initWithBytes:buf.ptr length:buf.size];
```

Comment:
You *could* prevent a copying using `-initWithBytesNoCopy:length:deallocator:`, but that's 10.9+ and I don't think we're that high yet.